substratevm/src/com.oracle.objectfile/src/com/oracle/objectfile/debugentry/Range.java

```diff
@@ -36,6 +36,7 @@
  */
 
 public class Range {
+    private final String compilationDirectory;
```

Comment:
This name seems wrong as the value is not actually anything to do with compilation. I understand that it is used to populate the comp_dir DWARF property but that use is incidental to its origin -- which I think it is more helpful to signal. So I'd prefer to use field name cachePath and modifying the getter accordingly.

Reply:
I agree. This should now be fixed all the way to `com.oracle.objectfile.elf.dwarf.DwarfInfoSectionImpl#writeCU`.